Makefile

```diff
@@ -17,11 +17,15 @@ help: ## Prints all the targets in all the Makefiles
 
 .PHONY: path_up_gateway
 path_up_gateway: ## Run just the PATH gateway without any dependencies
```

Comment:
Yes, there is a working implementation of the **entire** Envoy/PATH stack running in Tilt in PR #60 .

Once Arash's PR #58 is merged in I will reconcile that PR with `main` so I suggest we don't worry about anything related to Docker vs. Tilt for this PR with the understanding that it will be dealt with very shortly.

Reply:
@adshmh #58 is 🟢 and ✅ so PTAL when you can.